src/test/resources/org/wildfly/test/integration/vdx/transformations/messaging/InvalidTypeSetLongInServerIdInAcceptor.groovy

```diff
@@ -0,0 +1,9 @@
+attr = [name: 'in-vm']
+attr['server-id'] = '214748364700' // max int is just 2147483647
```

Comment:
this could be parametrized to avoid 2 almost the same groovy scripts (InvalidTypeForServerIdInAcceptor.groovy is the next one)

Look for example at https://github.com/jboss-eap-qe/vdx-wildfly-testsuite/pull/23/files#diff-a5ed37bbeb2ee034a8b2ca23fdf6bc3eR148